test/src/sdk-harness/test_artifacts/storage_vec/svec_u8/src/main.sw

```diff
@@ -0,0 +1,73 @@
+contract;
+
+use std::option::*;
+use std::result::*;
+use std::storage::{StorageVec, StorageVecError};
+
+abi MyContract {
+    #[storage(read, write)]
+    fn vec_u8_push(value: u8);
+    #[storage(read)]
+    fn vec_u8_get(index: u64) -> u8;
+    #[storage(read, write)]
+    fn vec_u8_pop() -> u8;
+    #[storage(read, write)]
+    fn vec_u8_remove(index: u64) -> u8;
+    #[storage(read, write)]
+    fn vec_u8_swap_remove(index: u64) -> u8;
+    #[storage(read, write)]
+    fn vec_u8_insert(index: u64, value: u8);
+    #[storage(read)]
+    fn vec_u8_len() -> u64;
+    #[storage(read)]
+    fn vec_u8_is_empty() -> bool;
+    #[storage(write)]
+    fn vec_u8_clear();
+}
+
+storage {
+    vec_u8: StorageVec<u8>,
```

Comment:
You could rename this to "my_vec" or something that isn't specific to this test. It still makes sense and it allows you to copy and paste it across other tests.